packages/flame/src/hooks/useEventListener.ts

```diff
@@ -5,21 +5,18 @@ type CallbackEvent<T> = (event: T) => void;
 export function useEventListener<T extends Event>(
   eventKey: string,
   callback: CallbackEvent<T>,
-  target?: EventTarget,
+  target: EventTarget = document,
 ) {
   React.useEffect(() => {
     const captureEvent: CallbackEvent<T> = event => {
       callback(event);
     };
```

Comment:
I might be missing something with how function declaration vs execution works but not sure this function is doing much but sitting here and looking pretty haha :)

Reply:
Maybe the original author thought it would take the last version of the callback. But it's a mistake. A ref as you suggested would solve that though.